controllers/organiser.go

```diff
@@ -0,0 +1,125 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/kossiitkgp/kwoc-backend/v2/middleware"
+	"github.com/kossiitkgp/kwoc-backend/v2/models"
+	"github.com/kossiitkgp/kwoc-backend/v2/utils"
+)
+
+type AcceptRejectProject struct {
+	// Id of the project in the database (required)
+	Id uint `json:"id"`
+	// Status to be set of the project
+	ProjectStatus bool `json:"project_status"`
+	// Status Remark to be set of the project
+	StatusRemark string `json:"status_remark"`
+}
+
+type ProjectOrg struct {
```

Comment:
Can this struct extend from an existing project struct?

Reply:
Imported from exsisting project struct